SOTS_2019_WDR.tex

```diff
@@ -0,0 +1,502 @@
+\end{checklist}
+\setcounter{checklistnum}{0}
+% end A2 section -- }}}comprises the final checks and tests performed on the Remote Launch Control System (RLCS) prior to rocket launch, as well as avionics systems arming
+
+% begin A3 section -- {{{
+\subsection{[A3] Abort Procedure - High Supply Pressure}
+\begin{checklist}
+    \item \primary{}: Close the Cylinder Valve.
+    \item \primary{}: Slowly open the Parallel Vent Valve.
+    \item \primary{}: Slowly open the Parallel Fill Valve.
+    \item \control{}: Confirm the following pressures are atmospheric:
+    \begin{checklist}
+        \item {[P1]} Supply pressure
+        \item {[P2]} Fill line pressure
+    \end{checklist}
+    \item \primary{}: Disarm the system:
+        \begin{checklist}    
+            \item Disconnect the ignition leads from the rocket.
+            \item Detatch the torsion springs from the disconnect mechanism.
+            \item Disarm the recovery electronics system using the magnetic switches.
+            \item Disarm the payload using the transponder.
+            \item Disconnect the fill line from the supply cylinder.
+            \item Replace the cap on the nitrous oxide supply cylinder.
+        \end{checklist}
+    \item \ops{}: Revisit cylinder cooling methods.
+\end{checklist}
+\setcounter{checklistnum}{0}
+% end A3 section -- }}}
+
+% begin A4 section -- {{{
+\subsection{[A4] Abort Procedure - Leak At Fill Plumbing}
+\begin{checklist}
+    \item \control{}: Close the Remote Fill Valve.
+    \item \control{}: Open the Tank Vent Valve.
+    \item \control{}: Open the Line Vent Valve.
+    \item \control{}: Confirm the following pressures are atmospheric:
+    \begin{checklist}
+        \item P2: Fill line pressure
+        \item P3: Rocket Tank pressure
+    \end{checklist}
+    \item \primary{} and \secondary{}: Return to plumbing setup
+    \item \primary{}: Close the Cylinder Valve.
+    \item \primary{}: Slowly open the Parallel Vent Valve.
+    \item \primary{}: Slowly open the Parallel Fill Valve.
+    \item \control{}: Confirm the following pressures are atmospheric:
+    \begin{checklist}
+        \item {[P1]} Supply pressure
+        \item {[P2]} Fill line pressure
+    \end{checklist}
+    \item \primary{}: Disarm the system:
+        \begin{checklist}    
+            \item Disconnect the ignition leads from the rocket.
+            \item Detatch the torsion springs from the disconnect mechanism.
+            \item Disarm the recovery electronics system using the magnetic switches.
+            \item Disarm the payload using the transponder.
+            \item Disconnect the fill line from the supply cylinder.
+            \item Replace the cap on the nitrous oxide supply cylinder.
+        \end{checklist}
+    \item \ops{}: Revisit plumbing setup.
+\end{checklist}
+\setcounter{checklistnum}{0}
+% end A4 section -- }}}
+
+% begin A5 section -- {{{
+\subsection{[A5] Abort Procedure - Remote Disconnect Failure}
```

Comment:
This procedure doesn't actually get called anywhere. It should get called right around the same time we actuate disconnect, if it's been unsuccessful.